netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
@mapuri 

> looks like netctl will only return netmaster's version and not netplugin's version or the docker/k8s plugin's version.

true; as in my comment in the high level comments for the PR; I am not addressing netplugin version for the reason that if I should reconcile and make it common across all and produce one such version.

How about if we do following:
- generate a version for contivk8s and show its version alongside `netplugin -v` output
- generate a version for netctl and show it alongside `netctl version`
- I am not sure if we should show versions of all netplugins in `netctl version` output; note that incompatible versions can work alongside as long as API versions match. Should we continue discussing the versioning mismatch between netmaster and netplugin as we discuss how to version the APIs. 

WDYT @shaleman, @erikh, @unclejack @jojimt